### ISSUE TITLE
[1.12.2] World Load crash with Galacticraft

### ISSUE THREAD

**Issue by brisingraerowing** [Icedice9/OreSheep#4]
```text
 Exception caught during firing event net.minecraftforge.event.world.WorldEvent$Load@cd63429:
java.lang.ArrayIndexOutOfBoundsException: -3
	at micdoodle8.mods.galacticraft.core.blocks.BlockBasic$EnumBlockBasic.byMetadata(BlockBasic.java:70) ~[BlockBasic$EnumBlockBasic.class:?]
	at micdoodle8.mods.galacticraft.core.blocks.BlockBasic.func_176203_a(BlockBasic.java:231) ~[BlockBasic.class:?]
	at oresheepmod.OreRegistryDraw.addAllSubBlocks(OreRegistryDraw.java:662) ~[OreRegistryDraw.class:?]
	at oresheepmod.OreRegistryDraw.getAllOres(OreRegistryDraw.java:346) ~[OreRegistryDraw.class:?]
	at oresheepmod.OreRegistryDraw.readConfig(OreRegistryDraw.java:166) ~[OreRegistryDraw.class:?]
	at oresheepmod.OreSheepEventHandler.onWorldLoad(OreSheepEventHandler.java:63) ~[OreSheepEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_2022_OreSheepEventHandler_onWorldLoad_Load.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:179) [EventBus.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71247_a(IntegratedServer.java:132) [chd.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71197_b(IntegratedServer.java:156) [chd.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:484) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_151]
```

Got this while getting ready to work on a technical mod pack I am building.

**Comment by mattcloward** [comment]
Thanks for the bug report! Are you using the 1.7.10 version or the 1.12.2 version? I'll have a fix out shortly!

**Comment by brisingraerowing** [comment]
1.12.2.

**Comment by mattcloward** [comment]
Fixed in v4.0.1
